velox/docs/develop/operators.rst

```diff
@@ -212,6 +213,33 @@ aggregate functions.
 Finally, note that computing measures over sorted input is only possible if
 aggregation step is 'single'. Such computations cannot be split into partial + final.
 
+.. _ExpandNode:
+
+ExpandNode
+~~~~~~~~~~~
+
+Apply all of the grouping expressions to every input row, hence we will get multiply output rows
```

Comment:
This documentation needs updating. Let's come up with a good class-level comment for ExpandNode first, then update this doc.

Reply:
The comments for the ExpandNode in the code are good now. Would you take a stab at re-writing this doc? Make sure not to use images. Look at the documentation for GroupId node for examples.